packages/toolpad-core/src/Account/Account.tsx

```diff
@@ -1,39 +1,67 @@
 import * as React from 'react';
+import { styled } from '@mui/material/styles';
 import PropTypes from 'prop-types';
-import { AvatarProps } from '@mui/material';
-import IconButton, { IconButtonProps } from '@mui/material/IconButton';
-import Button, { ButtonProps } from '@mui/material/Button';
 import Popover from '@mui/material/Popover';
-import Stack from '@mui/material/Stack';
-import Typography from '@mui/material/Typography';
-import LogoutIcon from '@mui/icons-material/Logout';
-import { SessionContext, AuthenticationContext } from '../AppProvider/AppProvider';
+import Divider from '@mui/material/Divider';
+import Button, { ButtonProps } from '@mui/material/Button';
+import IconButton, { IconButtonProps } from '@mui/material/IconButton';
+import Tooltip from '@mui/material/Tooltip';
+import Logout from '@mui/icons-material/Logout';
+import { Typography } from '@mui/material';
 import { SessionAvatar } from './SessionAvatar';
+import { SessionContext, AuthenticationContext } from '../AppProvider/AppProvider';
+import DEFAULT_LABELS from '../shared/labels/en';
+
+const AccountInfoContainer = styled('div')(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'row',
+  justifyContent: 'space-between',
+  padding: theme.spacing(2),
+  gap: theme.spacing(4),
+}));
+
+const SignOutContainer = styled('div')(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'row',
+  padding: theme.spacing(1),
+  justifyContent: 'flex-end',
+}));
 
 export interface AccountProps {
   /**
-   * Props to pass to the constituent components in the Account component.
-   * @default {}
-   * @example { signInButton: { color: 'primary' }, signOutButton: { color: 'secondary' } }
+   * The components used for each slot inside.
+   */
+  slots?: {
```

Comment:
You need to export the Slots in their own interface to automatically generate the documentation for them.
Take a look at https://github.com/mui/mui-toolpad/blob/master/packages/toolpad-core/src/DashboardLayout/DashboardLayout.tsx, line 253, should be a simple change!